frontend/src/cards/ui/MultiMapDialog.tsx

```diff
@@ -126,6 +129,11 @@ export function MultiMapDialog(props: MultiMapDialogProps) {
       ref={screenshotTargetRef}
     >
       <DialogContent dividers={true}>
+      <CardOptionsMenu
```

Comment:
probably dont need to fix for this PR, but interestingly the screenshot doesn't do the entire div, only the section that is visible 

![image](https://github.com/SatcherInstitute/health-equity-tracker/assets/41567007/c03e12a4-18ed-424e-a7af-c66df3fe05c2)

Reply:
Yeah, I noticed this too. I'll create an issue for it.